packages/stylelint-config-airbnb/index.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  plugins: [
+    'stylelint-scss',
+  ],
+  rules: {
+    'at-rule-empty-line-before': 'never',
```

Comment:
I'll reiterate my previous comment, it's late, its 3am and much wine has passed 😬 

This is **not** an ESLint configuration file, this is a stylelint configuration file

This repo has nothing to do with ESLint (aside ESLint lints the codebase)